keyboards/keebio/iris/keymaps/merrittlj/keymap.c

```diff
@@ -0,0 +1,156 @@
+// Copyright 2023 Danny Nguyen (@nooges)
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#include QMK_KEYBOARD_H
+
+#define _DVORAK 0
+#define _SHIFT 1
+#define _SYMBOL 2
+#define _FUNCTION 3
+#define _ADDITIONAL 4
```

Comment:
```suggestion
enum layers {
    _DVORAK,
    _SHIFT,
    _SYMBOL,
    _FUNCTION,
    _ADDITIONAL,
};
```

Reply:
I thought I had something like that.. I will fix that.